<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="addPointLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addPointButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add point </string>
    </property>
   </widget>
   <widget class="QWidget" name="drawPlace" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbRectangle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>104</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Прямоугольник</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPoint">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>104</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Точка</string>
    </property>
   </widget>
   <widget class="QLabel" name="dialogLable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите, для какой фигуры считывать значения </string>
    </property>
   </widget>
   <widget class="QLabel" name="rectangleOutputLable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>110</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pointsOutputLable">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>110</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вершины прмоугольника</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Точки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePointButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>delete point</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addPointLineEdit_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="drawButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>visualize</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
